tests/features/maxlength.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Check the feature Maxlength
+  In order to check the functionality of limiting and validating their maximum length in the edit form.
```

Comment:
The intention of the feature description is to describe which goal the feature has towards business.

The goal for business is not "to check the functionality of limiting and validating their maximum length". The limitation & validation of the maximum length is instead a way to reach this goal. We should probably ask the product owner what exactly their goal is so we can mention it in the feature file.